docs/pages/develop/user-interface/safe-areas.mdx

```diff
@@ -1,69 +1,114 @@
 ---
 title: Safe areas
-description: Learn how to add safe areas for your Expo project and other best practices.
+description: Learn how to add safe areas for screen components inside your Expo project.
 ---
 
+import { Collapsible } from '~/ui/components/Collapsible';
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
 import { Terminal, SnackInline } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
 
-Creating a safe area is a great way to ensure that your app's content is appropriately positioned around notches, status bars, home indicators, and other device and operating system interface elements.
+Creating a safe area ensures your app screen's content is positioned correctly. This means it doesn't get overlapped by notches, status bars, home indicators, and other interface elements. When the content gets overlapped, it gets concealed by these interface elements.
```

Comment:
```suggestion
Creating a safe area ensures your app screen's content is positioned correctly relative to notches, status bars, home indicators, and other interface elements that are controlled by the operating system or part of the physical hardware. Without a safe area, these interface elements may conceal your app screen's content.
```
I think it's important to emphasize these are device/OS elements outside of the developer's control. Also, that your positioning could be 100% correct relative to itself.

Reply:
I don't think it's a doubt that the might be concealed by these interface elements. It's always the case. Can you also elaborate on what you mean by " that your positioning could be 100% correct relative to itself"?